client/components/main/Desktop/CouncilSelector/index.js

```diff
@@ -21,40 +25,62 @@ const useStyles = makeStyles(theme => ({
 
 const CouncilSelector = ({
   councils,
+  selected,
+  unselected,
   updateCouncilsFilter,
+  updateSelectedCouncils,
```

Comment:
updateSelectedCouncils was declared in the imports section, is it a different updateSelectedCouncils?